Extension/src/githubAPI.ts

```diff
@@ -192,7 +194,8 @@ export function getTargetBuild(builds: Build[], userVersion: PackageVersion, upd
     }
     const latestVersionOnline: PackageVersion = new PackageVersion(builds[0].name);
     // Allows testing pre-releases without accidentally downgrading to the latest version
-    if (userVersion.isExtensionVersionGreaterThan(latestVersionOnline)) {
+    if ((!testingInsidersVsixInstall && userVersion.suffix && userVersion.suffix !== 'insiders') ||
```

Comment:
In this version, all the checks are performed in the `getTargetBuild `function. Previously, some of the checked were placed in `getTargetBuildInfo `function.